index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Apple Website</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://kit.fontawesome.com/89c0957ca7.js" crossorigin="anonymous"></script>
+
+
+</head>
+<body>
+    <main class="gen-grid">
+     <section class="top">
+        <nav>
+            <ul class="d-flex navbar-elements">
+                <li> <i class="fab fa-apple"> </i> </li>
+                <li>Store</li>
+                <li>Mac</li>
+                <li>iPod</li>
+                <li>iPhone</li>
+                <li>iPad</li>
+                <li>iTune</li>
+                <li>Support</li>
+                <li>
+                     <i id="search-icon" class="fas fa-search"></i>
+                     <input type="text">
+                </li>
+            </ul>
+        </nav>
+        <section class="img-text">
+          <img src = "img/text-1.png" alt="picture1"  >
+          <img src = "img/text-2.png" alt="picture1">
+          <img src = "img/text-3.png" alt="picture1">"
+        </section>
+      </section>
+      <section class="product-grid">
+        <img src = "img/iphone5s.jpg" alt="picture1"> 
+        <img src = "img/iphone5c.jpg" alt="picture1">
+        <img src = "img/happybirthdaymac.jpg" alt="picture1">
+        <img src = "img/supplier.jpg" alt="picture1">
+      </section>
```

Comment:
These should be links. The images should be links. [link to the page](https://web.archive.org/web/20140301004610/http://www.apple.com/)